src/stim/stabilizers/pauli_string.pybind.cc

```diff
@@ -382,6 +382,52 @@ void stim_pybind::pybind_pauli_string(pybind11::module &m) {
         )DOC")
             .data());
 
+    c.def(
+        "to_tableau",
+        [](const PyPauliString &self) {
+            return Tableau::from_pauli_string(self.value);
+        },
+        clean_doc_string(u8R"DOC(
+            Creates a Tableau equivalent to this Pauli string.
+
+            The tableau represents a Clifford operation that multiplies qubits
```

Comment:
Make a note that the sign of the Pauli operation is lost in the conversion.

Reply:
:+1: